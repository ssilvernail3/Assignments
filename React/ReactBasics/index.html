<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Shiba Inu</title>
</head>

<body>
    <h1>SHIBAS!!</h1>
    <div id="root"></div>
    <script src="https://unpkg.com/react/umd/react.development.js"></script>
    <script src="https://unpkg.com/react-dom/umd/react-dom.development.js"></script>
    <script src="https://unpkg.com/babel-standalone"></script>
    
    <script src="Shiba.js" type="text/jsx"></script>
    <script src="Sharpei.js" type="text/jsx"></script>
    <script src="RandonNum.js" type="text/jsx"></script>
    <script src="Animal.js" type="text/jsx"></script>
    <script src="App.js" type="text/jsx"></script>
</body>

</html>