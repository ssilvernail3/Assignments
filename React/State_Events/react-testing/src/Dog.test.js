import React from 'react';
import { render, screen } from '@testing-library/react';
import Dog from './Dog';

test('if renders without crahsing', () => {
    render(<Dog name="Rusty" isAdopted={true} />)
}); 

it('matches snapshot', () => {
    const { asFragment } = render(<Dog name="Rusty" isAdopted={true} />);
    expect(asFragment()).toMatchSnapshot();
});
it('matches snapshot', () => {
    const { asFragment } = render(<Dog name="Rusty" isAdopted={false} />);
    expect(asFragment()).toMatchSnapshot();
});