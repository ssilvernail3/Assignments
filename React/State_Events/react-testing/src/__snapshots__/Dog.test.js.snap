// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Dog"
  >
    <h1>
      Rusty
    </h1>
    <p>
      Rusty has been adopted!
    </p>
  </div>
</DocumentFragment>
`;

exports[`matches snapshot 2`] = `
<DocumentFragment>
  <div
    class="Dog"
  >
    <h1>
      Rusty
    </h1>
    <button>
      Adopt Me!!!!
    </button>
  </div>
</DocumentFragment>
`;
